service/src/event/order_events.rs

```diff
@@ -181,18 +187,30 @@ mod test {
         (after_1, after_2)
     }
 
+    /// Takes the given events from Recon and turns them into two vectors of insertable events.
```

Comment:
How are the two vectors different? Should we make a small struct to give them names?